packages/flutter_tools/bin/podhelper.rb

```diff
@@ -0,0 +1,126 @@
+# Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
what's the transition plan for add-to-app? Can we change this line in https://flutter.dev/docs/development/add-to-app/ios/project-setup#option-a---embed-with-cocoapods-and-the-flutter-sdk to load `File.join(ENV["FLUTTER_ROOT"], '.ios', 'Flutter', 'podhelper.rb')` or some such?

Reply:
Currently the CocoaPods add-to-app module lays down a similar but not quite the same podhelper in the ephemeral directory, which gets recreated on a `flutter clean`.  As long as we didn't rename those methods and break existing host Podfiles (unfortunately the names don't reference iOS or the module: `install_all_flutter_pods`, `install_flutter_engine_pod`, `install_flutter_plugin_pods`) we could transition them to this bin podhelper, but I don't know what it would really buy us (other than they would get updates instantly instead on a `clean`).  I'm concerned it would make the CI story more complicated because `flutter` would need to be available on every machine, even if the entire module and podspecs and binaries were copied in.

P.S. We wouldn't want them to need `FLUTTER_ROOT` on their path, it would have to be parsed from the Generated files the same way this one is.

Reply:
ya, it should be a separate task. 

What I had in mind actually may or may not work. One small ergonomic annoyance now is if someone checks out the iOS and flutter projects, it currently takes 3 steps to build again. Go to flutter, flutter pub get to make the .ios folder with podhelper.rb, then go to iOS, call pod install, then open xcode, run.

Perhaps we can save one step if the various flutter_install_* code called flutter pub get to make the underlying xcodeproj if needed etc?